tests/test_script.py

```diff
@@ -6,7 +7,8 @@
 
 DIR = os.path.dirname(os.path.abspath(__file__))
 REPO = os.path.dirname(DIR)
-WHITELIST = os.path.join(REPO, 'vulture', 'whitelists', 'stdlib.py')
+WHITELIST_GLOB = os.path.join(REPO, 'vulture', 'whitelists', '*.py')
+WHITELISTS = glob.glob(WHITELIST_GLOB)
```

Comment:
We have the following options, for just the test part:

* os.listdir
* glob.glob
* pkg_resources.resource_listdir

Do we have other alternatives? Which one's the best in your opinion?
Thanks!